config/services/bootp-server.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Note, this is the same as `config/services/dhcp.xml`. Just noting, I think it's fine to have both.